src/components/menu/components/dishes/components/categoryDishes/CategoryDishes.js

```diff
@@ -0,0 +1,62 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _startCase from 'lodash/startCase';
+import _noop from 'lodash/noop';
+import dishCategoryReader from './readers/dishCategoryReader';
+
+//Components
+import Dish from './components/dish';
+
+//CSS
+import './categoryDishes.css';
+
+function renderCategoryDish(dish, quantities, handleEvents) {
+  const id = dishCategoryReader.id(dish);
+  return (
+    <Dish
+      key={id}
+      dish={dish}
+      quantities={quantities}
+      handleEvents={handleEvents}
+    />
+  );
+}
+
+function renderCategoryDishes(dishes, quantities, handleEvents) {
+  return dishes.map((dish) =>
```

Comment:
- try to use functions from lodash like map, filter and all. 
- try to use use inline functions wherever map and filters are used